src/Content/Backend/Solution/Monaco.Template.Backend.Api/Auth/Scopes.cs

```diff
@@ -5,18 +5,18 @@ public static class Scopes
 {
 	public const string CompaniesRead = "companies:read";
 	public const string CompaniesWrite = "companies:write";
-#if filesSupport
-	public const string FilesRead = "files:read";
+#if !excludeFilesSupport
 	public const string FilesWrite = "files:write";
+	public const string ProductsWrite = "products:write";
 #endif
 
 	public static List<string> List => new()
```

Comment:
Completely optional, but can use the new collection expression syntax here to replace ` = new() { .. }` with ` = [ .. ]`

Reply:
Yeah, I keep forgetting about it because for some reason VS 2022 still screams error with that new syntax... Not sure if it's me or a more general issue.

Reply:
Ok, it was a problem on my side. Updated some stuff and now it's working. Already ported everything there was to the new syntax.